src/warpedview.jl

```diff
@@ -1,62 +1,86 @@
-immutable WarpedView{T,N,A<:AbstractArray,F1<:Transformation,F2<:Transformation,I} <: AbstractArray{T,N}
+_build_extrapolation(etp::AbstractExtrapolation) = etp
+
+function _build_extrapolation{T}(itp::AbstractInterpolation{T}, fill::FillType = _default_fill(T))
+    etp = extrapolate(itp, fill)
+    _build_extrapolation(etp)
+end
+
+function _build_extrapolation{T,N,D<:Union{Linear,Constant}}(parent::AbstractArray{T,N}, degree::D = Linear(), args...)
+    itp = Interpolations.BSplineInterpolation{T,N,typeof(parent),BSpline{D},OnGrid,0}(parent)
+    _build_extrapolation(itp, args...)
+end
+
+immutable WarpedView{T,N,A<:AbstractArray,F1<:Transformation,I,F2<:Transformation,E<:AbstractExtrapolation} <: AbstractArray{T,N}
     parent::A
     transform::F1
-    transform_inv::F2
     indices::I
+    inverse::F2
```

Comment:
Maybe `FwdWarpedView` and `BwdWarpedView`? Although I do think that one of them should just be `WarpedView`. Personally I vote for keeping the current one as is, and adding the alternative with one of the following names (any other ideas?)

- `InvWarpedView`
- `BwdWarpedView`
- `IdsWarpedView`
- `WarpedIndicesView`